README.md

```diff
@@ -1,73 +1,56 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+## Capastone1 Project
 
-> Description the project.
+This project is to help demostrating or put into practice what we have learned at Microverse for the time we spent at Microverse program until now. Please have a look at my Live site and give me feedback Live Demo. 🙂
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSSfunction
+- Javascript
 
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+## Live Demo
+[Live Demo Link](https://sarazatoussaint.github.io/Grill_Capstone-1/)
 
 
 ## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+git clone (https://github.com/sarazaToussaint/Grill_Capstone-1.git)
 
 
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
+The basic requirements for building the executable are:
 
-### Setup
+- A working browser application (Google chrome, Mozilla Fire fox, Microsoft edge ...)
+- VSCode or any other equivalent code editor
+- Node Package Manager (For running, debuging, or installing dependancies to be used)
 
 ### Install
-
-### Usage
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
 
 ### Run tests
+- npx eslint.
+- npx webhint .
+- npx stylelint "**/*.{css,scss}"
 
-### Deployment
-
-
-
-## Authors
+### Documentation
+[Loom Video Link] (https://www.loom.com/share/4d2140998d4e4804936ec8408637fe35)
```

Comment:
Congratulations for describing the project and showing a demo of the project features (different pages and different screen sizes)

But the project requirements state that the video should also include a highlight of some interesting piece of code or something you built that you are very proud of.

- [ ] Kindly, rerecord the video and besides the presentation of your project and the description of it, including a highlight of some interesting piece of code or something you built that you are very proud of.